e2e/utils/flows/deleteDashboard.ts

```diff
@@ -29,6 +29,7 @@ export const deleteDashboard = ({ quick = false, title, uid }: DeleteDashboardCo
 
 const quickDelete = (uid: string) => {
   cy.request('DELETE', fromBaseUrl(`/api/dashboards/uid/${uid}`));
+  cy.request('DELETE', fromBaseUrl(`/api/dashboards/uid/${uid}/trash`));
```

Comment:
Why do you need to add this although `deleteDashboard()` is not used in e2e/dashboards/TestRestoreDashboard.json?

Reply:
There is a panels test that creates a dashboard at the start of each test and deletes it at the end using this method. We need to permanently delete the dashboard so the next test can reimport it